frontend/src/v5/services/actionsDispatchers.ts

```diff
@@ -53,4 +53,4 @@ export const TicketsActionsDispatchers = createActionsDispatchers<ITicketsAction
 export const TicketsCardActionsDispatchers = createActionsDispatchers<ITicketsCardActionCreators>(TicketsCardActions);
 export const TicketCommentsActionsDispatchers = createActionsDispatchers<ITicketCommentsActionCreators>(TicketCommentsActions);
 export const UsersActionsDispatchers = createActionsDispatchers<IUsersActions>(UsersActions);
-export const ViewerActionsDispatchers = createActionsDispatchers<ViewerActionCreators>(ViewerActions);
+export const ViewerActionsDispatchers = createActionsDispatchers<ViewerActionsCreatorsType>(ViewerActionsCreators);
```

Comment:
Why were these renamed in this way? Would it not be best to be consistent with the naming convention in this file of IViewerActionCreators and ViewerActions respectfully? Although I can see that not all the other dispatchers are consistent with this